src/views/Report/ReportFigureTable/index.tsx

```diff
@@ -0,0 +1,263 @@
+                (item) => item.createdBy?.fullName,
+                { sortable: true },
+            ),
+            createTextColumn<ReportFigureFields, string>(
+                'country_name',
+                'Country Name',
+                (item) => item.country?.name,
+                { sortable: true },
+            ),
+            // createLinkColumn<ReportFigureFields, string>(
+            //    'entry__event__crisis__name',
+            //    'Crisis',
+            //    (item) => ({
+            //        title: item.entry.event?.crisis?.name,
+            //        attrs: { crisisId: item.entry.event?.crisis?.id },
+            //    }),
+            //    route.crisis,
+            //    { sortable: true },
+            //  ),
+            createLinkColumn<ReportFigureFields, string>(
+                'entry__event__name',
+                'Event',
+                (item) => ({
+                    title: item.entry.event?.name,
+                    // FIXME: this may be wrong
+                    attrs: { eventId: item.entry.event?.id },
+                }),
+                route.event,
+                { sortable: true },
+            ),
+            createLinkColumn<ReportFigureFields, string>(
+                'entry__article_title',
+                'Entry',
+                (item) => ({
+                    title: item.entry.articleTitle,
+                    attrs: { entryId: item.entry.id },
+                }),
+                route.entryView,
+                { cellAsHeader: true, sortable: true },
+            ),
+            createTextColumn<ReportFigureFields, string>(
+                'entry__term',
+                'Term',
+                (item) => item.term?.name,
+                { sortable: true },
+            ),
+            createTextColumn<ReportFigureFields, string>(
+                'entry__role',
+                'Role',
+                (item) => item.role,
+                { sortable: true },
+            ),
+            createTextColumn<ReportFigureFields, string>(
+                'figure_type',
+                'Figure Type',
+                (item) => item.category?.type,
+                { sortable: true },
+            ),
+            createNumberColumn<ReportFigureFields, string>(
+                'total_figures',
+                'Total Figures',
+                (item) => item.totalFigures,
+                { sortable: true },
+            ),
+            createNumberColumn<ReportFigureFields, string>(
+                'reported_figures',
+                'Reported Figures',
+                (item) => item.reported,
+                { sortable: true },
+            ),
+            createDateColumn<ReportFigureFields, string>(
+                'started_at',
+                'Start Date',
+                (item) => item.startDate,
+                { sortable: true },
+            ),
+            createDateColumn<ReportFigureFields, string>(
+                'ended_at',
```

Comment:
The name is not correct,
```suggestion
                'end_date',
```